stetho/src/main/java/com/facebook/stetho/inspector/elements/android/ViewHighlighter.java

```diff
@@ -58,7 +59,10 @@ public void setHighlightedView(View view, int color) {
     //       causing every single view to allocate a ViewOverlay
 
     private final Handler mHandler;
-    private final ColorDrawable mHighlightDrawable;
+    private ColorDrawable[] mHighlightDrawables;
+
+    private static final int MARGIN_OVERLAY_COLOR = Color.parseColor("#aaf4ca9e");
```

Comment:
Prefer not using `Color.parseColor` since the memory layout of colors is well defined: `0xaaf4ca9e` is the value :)